content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,124 @@
-====================================================================
-When Should you Use Packages, Units of Measure or Special Packaging?
-====================================================================
+==========================================
+Units of measure, packages, and packagings
+==========================================
 
-Units of Measure
-================
-
-Units of measure specify the unit used to handle a product. In Odoo, you
-have the possibility to specify the unit of measure in which you manage
-your stock and the one which is used when purchasing the product to your
-supplier.
+In Odoo, there are a variety of ways to specify the amount of products being bought, stocked, and
+sold. Units of measure, packages, and packagings are all available to streamline inventory flows,
+allowing for a variety of configurations for products as they enter and leave the warehouse.
 
-.. image:: usage/usage_01.png
-    :align: center
-
-The *conversion* between the different units of measure is done
-automatically. The only condition is that all the units have to be in
-the *same category* (Unit, Weight, Volume, Length,...)
+Units of measure
+================
 
-For example, if I have the following reordering rule for the egg and I
-run the scheduler, the quantity added in the automatically generated
-purchase order will be in dozens but what will enter the stock will be
-units.
+*Units of measure* refers to different measurable quantities used to handle products, such as units,
+weight, time, size, etc. For example, different units of measure for weight can be kilos, pounds,
+ounces, grams, etc. In Odoo, stock is easily managed and products are more efficiently purchased
+from suppliers by specifying different units of measure for both purchasing and selling.
 
-.. image:: usage/usage_02.png
-    :align: center
+.. image:: usage/rope-14.png
+   :align: center
+   :alt: Specify unit of measure for selling a product vs purchasing.
 
-.. image:: usage/usage_03.png
-    :align: center
+Once a product has a :guilabel:`Unit of Measure` and a :guilabel:`Purchase UoM` set on the product
+form, Odoo will automatically convert the different units in the product's purchase/sales orders and
+the orders' respective delivery orders/receipts. The only condition is that all of the units have to
+be in the *same category* (Unit, Weight, Volume, Length, etc.). For example, a product can have its
+:guilabel:`Unit of Measure` field set to `feet (ft)` and its :guilabel:`Purchase UoM` set to
+`centimeters (cm)`. When a :abbr:`PO (purchase order)` is created for that product, it will list the
+quantity in centimeters. Then, when the :abbr:`PO (purchase order)` is confirmed, Odoo automatically
+generates a receipt and converts the centimeters to feet. The receipt will list the quantity in
+feet.
```

Comment:
@ksc-odoo, thanks again for such an in-depth review!

To better understand our team's documentation practices, I wanted to understand your reasoning for choosing to italicize **units of measure** and a **Purchase UoM**, as opposed to setting them as guilabels?

Reply:
> Once a product has a _units of measure_ and a _Purchase UoM_ set on the product form, Odoo automatically converts the different units in the product's purchase/sales orders and the orders' respective delivery orders/receipts.
> 
> The only condition is that all of the units have to be in the _same category_ (Unit, Weight, Volume, Length, etc.).
> 
> For example, a product can have its :guilabel:`Unit of Measure` field set to `feet (ft)` and its :guilabel:`Purchase UoM` set to `centimeters (cm)`. When a :abbr:`PO (purchase order)` is created for that product, it will list the quantity in centimeters.
> 
> Then, when the :abbr:`PO (purchase order)` is confirmed, Odoo automatically generates a receipt, and converts the centimeters to feet. The receipt will list the quantity in feet.

Slight adjustments
```

Once a product has default :abbr:`UoM (Unit of Measure)` and Purchase UoM values set on the product
form, Odoo automatically converts the different units in the product's purchase/sales orders and
the orders' respective delivery orders/receipts.

The only condition is that all of the units have to be in the same category such as unit, weight,
volume, length, etc.

For example, a product can have its :guilabel:`Unit of Measure` field set to feet (ft) and its
:guilabel:`Purchase UoM` set to centimeters (cm). When a :abbr:`PO (Purchase Order)` is created for
that product, it will list the quantity in centimeters.

Then, when the :abbr:`PO (Purchase Order)` is confirmed, Odoo automatically generates a receipt, and
converts the centimeters to feet. The receipt will list the quantity in feet.

```

Reply:
> _NOTE_ - would it be beneficial to the doc to briefly explain how a user can set a Unit of Measure and Purchase UoM on the product form?
> Also, perhaps an ".. example::" tag could be utilized here. Maybe even consider adding a screenshot to supplement the example being explained...

@ksc-odoo 
This is the change I've added based on your suggestion!

```
.. example::
   The product, `Rope` has its :guilabel:`Unit of Measure` field set to feet (ft) and its
   :guilabel:`Purchase UoM` set to centimeters (cm). When creating a :abbr:`PO (Purchase Order)`
   for rope, the Purchase UoM, centimeters, appears on the form to measure the quantity of rope the
   company buys from the vendor.

   .. image:: usage/purchase-rope-in-cm.png
      :align: center
      :alt: Purchase order for rope in cm

   Then, when the :abbr:`PO (Purchase Order)` is confirmed, Odoo automatically generates a receipt
   and converts the centimeters to feet. The receipt lists the quantity in feet.

   .. image:: usage/receive-rope-in-ft.png
      :align: center
      :alt: Warehouse reception of rope in ft
```